### PR TITLE
chore: add husky and lint-staged

### PR DESCRIPTION
**Pull request by v61265**
### 描述
- 參考 [該篇文章](https://jenniesh.github.io/dev/NPM/husky-lint-staged/) 設定 husky 和 lint-staged。

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn lint-staged
```
Comment (erase2004) [inline .husky/pre-commit]: 用 `npx lint-staged` 就好了

File: package.json [hunk 2]
```diff
@@ -14,7 +14,14 @@
     "lint-staged": "^13.0.3",
     "prettier": "^2.7.1"
   },
+  "scripts": {
+    "prepare": "husky install"
+  },
   "lint-staged": {
-    "*.{js,jsx}": "eslint --cache --fix"
+    "*.{ts,tsx,js,jsx}": "eslint --cache --fix",
+    "*.{html,scss,css,vue}": [
```
Comment (erase2004) [inline package.json]: `.vue` 不會用到吧

File: package.json [hunk 3]
```diff
@@ -14,7 +14,14 @@
...
+      "prettier --write",
+      "stylelint --fix"
```
Comment (erase2004) [inline package.json]: stylelint 沒有用到

File: packages/mirror-tv-next/package.json [hunk 4]
```diff
@@ -35,6 +35,8 @@
     "@types/styled-components": "^5.1.34",
     "eslint": "^8",
     "eslint-config-next": "14.0.3",
+    "husky": "^9.0.11",
```
Comment (erase2004) [inline packages/mirror-tv-next/package.json]: package 不用安裝這兩個

**Review by erase2004 — CHANGES_REQUESTED** [review]

File: .husky/pre-commit (hunk 1 above)
Comment (v61265) [reply .husky/pre-commit]: 感謝提醒，已於 [8a74011](https://github.com/mirror-tv/Tabris/pull/70/commits/8a74011f1a96f7f06025612679c81751700a22cb) 統一修改

File: package.json (hunk 2 above)
Comment (v61265) [reply package.json]: 對齁！感謝提醒，已於 [8a74011](https://github.com/mirror-tv/Tabris/pull/70/commits/8a74011f1a96f7f06025612679c81751700a22cb) 統一修改

File: package.json (hunk 3 above)
Comment (v61265) [reply package.json]: 感謝提醒，已於 [8a74011](https://github.com/mirror-tv/Tabris/pull/70/commits/8a74011f1a96f7f06025612679c81751700a22cb) 統一修改

File: packages/mirror-tv-next/package.json (hunk 4 above)
Comment (v61265) [reply packages/mirror-tv-next/package.json]: 感謝提醒，已於 [8a74011](https://github.com/mirror-tv/Tabris/pull/70/commits/8a74011f1a96f7f06025612679c81751700a22cb) 統一修改

**Review by erase2004 — APPROVED** [review]
LGTM 👏

**Pull request merged**